src/services/utilities/is-valid-coords-service.js

```
@@ -0,0 +1,66 @@
+const LATITUDE_MIN = -90;
+const LATITUDE_MAX = 90;
+const LONGITUDE_MIN = -180;
+const LONGITUDE_MAX = 180;
+
+
+export const isValidCoordsService = {
+
+    getCoords(param1, param2) {
+        // 'lat, lng'
+        if (typeof param1 === 'string' && typeof param2 === 'undefined') {
+            [param1, param2] = param1.split(',');
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)